src/main/webapp/renderers/Step2Renderer.js

```diff
@@ -0,0 +1,59 @@
+import RendererInterface from './RendererInterface.js';
+
+export default class Step2Renderer extends RendererInterface {
+    constructor(parentElement) {
+        super(parentElement);
+    }
+
+    generateInfoPane(prevEvent, nextEvent, index) {
+        if(prevEvent.location == nextEvent.location) {
+            return `
```

Comment:
Nit: Could we make this string a constant with a meaningful name?

Same idea could be applied below

Reply:
Alright, I will combine them into a nice ?: operator!

Reply:
Done! Can you have a look?